libs/ddhub-client-gateway-message-broker/src/lib/handler/relogin.handler.ts

```diff
@@ -0,0 +1,17 @@
+import { CommandHandler, ICommandHandler } from '@nestjs/cqrs';
+import { ReloginCommand } from '../command/relogin.command';
+import { Logger } from '@nestjs/common';
+import { DdhubLoginService } from '../services';
+
+@CommandHandler(ReloginCommand)
+export class ReloginHandler implements ICommandHandler<ReloginCommand> {
+  protected readonly logger = new Logger();
+
+  constructor(protected readonly ddhubLoginService: DdhubLoginService) {}
+
+  public async execute(): Promise<void> {
+    await this.ddhubLoginService.login().catch((e) => {
```

Comment:
Is re-login just calling `login()` or should it call `logout()` first?

Reply:
there is no logout functionality in case of did auth proxy